ros/src/computing/perception/detection/vision_detector/packages/vision_ssd_detect/CMakeLists.txt

```diff
@@ -25,6 +28,7 @@ catkin_package(
         sensor_msgs
         std_msgs
         autoware_msgs
```

Comment:
This can be removed and add the following in `add_dependencies()`:

```
    add_dependencies(vision_ssd_detect
            ${autoware_msgs_generate_messages_cpp}
            ${autoware_config_msgs_generate_messages_cpp}
            ${catkin_EXPORTED_TARGETS}
            )
```

The reason for not using the `find_package(catkin REQUIRED COMPONENTS ...)` idiom is that it was removed in ROS2, and given that we're now adding colcon, I thought it'd save us some time in the future in case we wanted to add ROS2.

Reply:
I see. However, for some reason, it won't build on catkin unless the messages are added to the `catkin REQUIRED COMPONENTS`

Reply:
The good news is that the problematic packages are marked as `deprecated`, and will be removed with  #1664

Reply:
In that case... awesome :-)